lib/take_away.rb

```diff
@@ -0,0 +1,56 @@
+class TakeAway
+ 
+
+  attr_writer :menu, :meal_order
```

Comment:
Always be careful to use `attr_writer` and do not use it if you do not need it (you do not need it in your class because you have defined methods to access the properties of your class). You always want to restrict the access that the outside world has to the property using your own methods. As I have shown you I could do something like 
```
restaurant = TakeAway.new
restaurant.menu = nil
``` 
And this will change your menu to nil. `attr_reader` is less powerful BUT I could still use any method on your properties and do something like `restaurant.meal_order.replace([1, 2, 3])` and this would work